packages/compiler/src/server/type-details.ts

```diff
@@ -62,11 +63,19 @@ function getSymbolDocumentation(program: Program, symbol: Sym) {
   }
 
   // Add @doc(...) API docs
-  const type = symbol.type ?? program.checker.getTypeForNode(symbol.declarations[0]);
-  const apiDocs = getDocData(program, type);
-  // The doc comment is already included above we don't want to duplicate. Only include if it was specificed via `@doc`
-  if (apiDocs && apiDocs.source === "decorator") {
-    docs.push(apiDocs.value);
+  let type = symbol.type;
+  if (!type) {
+    const entity = program.checker.getTypeOrValueForNode(symbol.declarations[0]);
+    if (entity && isType(entity)) {
+      type = entity as Type;
```

Comment:
```suggestion
      type = entity;
```

not sure you should need this `as Type` as you already check for that above

Reply:
removed.